src/python_inspector/utils_pypi.py

```diff
@@ -341,6 +360,15 @@ def sorted(cls, namevers):
         return sorted(namevers or [], key=cls.sortable_name_version)
 
 
+class Link(NamedTuple):
+    """
+    A link to a sdist/wheel on PyPI.
+    """
+
+    url: str
+    required_python: str
```

Comment:
is this a specifier string? And why not keep the original "python_requires" name?